src/siembol/templates/_helpers.tpl

```diff
@@ -0,0 +1,196 @@
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+
+{{- define "siembol.alerting.deploy.topology.name" -}}
+{{- printf "%s-%s" (include "siembol.name" .) .Values.alerting.deploy.topology.name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "siembol.alerting.deploy.topology.fullname" -}}
+{{- $name := default .Chart.Name .Values.alerting.deploy.topology.name -}}
+{{- printf "%s-%s" (include "siembol.fullname" .) $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a fully qualified alerting deploy rules name and fullname.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+
+{{- define "siembol.alerting.deploy.rules.name" -}}
+{{- printf "%s-%s" (include "siembol.name" .) .Values.alerting.deploy.rules.name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "siembol.alerting.deploy.rules.fullname" -}}
+{{- $name := default .Chart.Name .Values.alerting.deploy.rules.name -}}
+{{- printf "%s-%s" (include "siembol.fullname" .) $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Common labels
+*/}}
+{{- define "siembol.labels" -}}
+helm.sh/chart: {{ include "siembol.chart" . }}
+{{ include "siembol.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end -}}
+
+{{/*
+Selector labels
+*/}}
+{{- define "siembol.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "siembol.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "siembol.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create -}}
+    {{ default (include "siembol.fullname" .) .Values.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Set the nimbus name for the Storm chart
+*/}}
+{{- define "storm.nimbus.fullname" -}}
+{{- printf "%s-%s" .Release.Name "storm-nimbus" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Set the ZooKeeper server for the siembol chart
+*/}}
+
+{{- define "siembol.zookeeper.server" -}}
+{{- if .Values.zookeeper.enabled -}}
+{{- printf "%s-%s" .Release.Name "zookeeper" | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $nullcheck := required "If not using the siembol chart's built-in Zookeeper (i.e. `.Values.zookeeper.enabled: false`), `.Values.zookeeper.servers` is required" .Values.zookeeper.servers -}}
+{{- tpl (.Values.zookeeper.servers | toYaml) $ -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Set the name for the Kafka chart
+*/}}
+{{- define "kafka.fullname" -}}
+{{- printf "%s-%s" .Release.Name "kafka" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create the config for the alerting Storm topology
+*/}}
+{{- define "siembol.alerting.config" -}}
+{
+  "config_version": 1,
```

Comment:
this should be in siembol-config git repo as an admin config

Reply:
Indeed, it looks like it is:

* https://github.com/G-Research/siembol-config/blob/main/alerts/admin_config/admin_config.json